tools/labs/templates/kernel_modules/3-error-mod/err_mod.c

```diff
@@ -1,6 +1,5 @@
 #include <linux/init.h>
 #include <linux/kernel.h>
-/* TODO: add missing kernel headers */
 #include <linux/module.h>
```

Comment:
Are you sure this is the correct fix? We intentionally added the TODO there. This exercises shows the reader what happens if the proper headers are not included.

Thanks for the contribution! But I think you should rethink your solution. If there is a stray character before the comment just remove that but please leave the comment there.

Reply:
Its a wrong fix definetly and I wasn't aware if it was an exercise, as I was also getting build failure in doing make docs :p

https://github.com/sphinx-doc/sphinx/issues/6269

As I am getting my senses back I see this is wrong solution I have posted.

Reply:
Only thing that bothers me here is without using the comments I can compile it easily. So what headers we need, as error says nothing about header missing.